vespalib/src/vespa/vespalib/net/tls/transport_security_options_reading.cpp

```diff
@@ -0,0 +1,102 @@
+// Copyright 2018 Yahoo Holdings. Licensed under the terms of the Apache 2.0 license. See LICENSE in the project root.
+#include "transport_security_options_reading.h"
+#include <vespa/vespalib/data/slime/slime.h>
+#include <vespa/vespalib/util/exceptions.h>
+#include <vespa/vespalib/io/fileutil.h>
+#include <vespa/vespalib/io/mapped_file_input.h>
+#include <vespa/vespalib/data/memory_input.h>
+
+namespace vespalib::net::tls {
+
+/*
+
+ Proposed JSON format for TLS configuration file:
+
+{
+  "files": {
+    "private-key": "myhost.key",
+    "ca-certificates": "my_cas.pem",
+    "certificates": "certs.pem"
+  },
+  // for later:
+  "peer-taggers": [
+    {
+      "requirements":[
+        {
+          "field": "SAN"
+          "must-match": "DNS:foo.bar.baz.*"
+        }
+      ],
+      "tags": ["cluster-peers", "config-server"] // or "roles"? Avoid ambiguities with Athenz concepts
+    },
+    {
+      "requirements":[
+        { "field":"CN", "must-match": "config.blarg.*"}
+      ],
+      "tags": ["config-server"]
+    }
+  ]
+}
+
+ */
+
+using namespace slime;
```

Comment:
you might get away with just using the 'slime::convenience' namespace here

Reply:
Fixed